src/components/Books.module.css

```diff
@@ -0,0 +1,119 @@
+.progCircle {
+  width: 5.625rem;
+  max-height: 5.625rem;
+}
+
+.smContainer {
+  position: relative;
+  bottom: 100px;
+  left: 600px;
+  display: flex;
+  flex-direction: row;
+}
+
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap");
```

Comment:
Please add all the @import rules at the top of the document, you can find why and more information on this [link](https://www.w3schools.com/cssref/pr_import_rule.asp).